pkg/preparer/orchestrate.go

```diff
@@ -218,6 +220,22 @@ func (p *Preparer) handlePods(podChan <-chan ManifestPair, quit <-chan struct{})
 	}
 }
 
+// This slice is eventually serialized into YAML, so we're careful about escaping
+func logBridgeExec(pod *pods.Pod) []string {
+	return []string{
+		pod.P2Exec,
+		"-u",
+		"nobody",
+		"-e",
+		pod.EnvDir(),
+		"--",
+		filepath.Join(os.Getenv("POD_HOME"), "p2-log-bridge", "current", "bin", "p2-log-bridge"),
+		"bridge",
+		"-logExec",
+		svlogdExec + " ", // trailing space forces single-quotes by YAML serializer
```

Comment:
Looks like we don't. I had assumed we needed to say `'svlogd -tt ./main'` where the single quotes were necessary. I've just tested this hypothesis and found the quotes to be unnecessary. Said another way the following is valid configuration for our implementation of servicebuilder.

```
…
  log:
  - /usr/local/bin/p2-exec
  - -u
  - nobody
  - -e
  - /data/pods/slug/env
  - --
  - /data/pods/p2-preparer/p2-log-bridge/current/bin/p2-log-bridge
  - bridge
  - -logExec
  - svlogd -tt ./main
```